pkg/sentry/platform/systrap/README.md

```diff
@@ -0,0 +1,33 @@
+# The systrap platform
+
+This platform is similar with the ptrace platform with the difference how system
```

Comment:
What are pros and cons compared to ptrace?

Reply:
Systrap has had less battle-testing than ptrace, but has the potential to handle system calls quicker than ptrace does. Also, like ptrace, it doesn't require virtualization support (whereas gVisor's KVM platform does).
If you have a sandboxed workload that is using ptrace now, try out Systrap! It'd be good to hear about your experience in terms of reliability, performance, and resource usage.